src/map_preview_graphic.c

```diff
@@ -54,143 +54,706 @@ typedef struct UnkStruct_0206A388 {
-     .unused3C = 22 },
-    { // Cerulean Cave
-        .mapId    = MAP_D03R0101,
-     .unused4  = 0,
-     .files    = { { .palNo = NARC_preview_graphic_preview_graphic_cerulean_cave_morning_NCLR, .charNo = NARC_preview_graphic_preview_graphic_cerulean_cave_morning_NCGR_lz, .scrnNo = NARC_preview_graphic_preview_graphic_cerulean_cave_morning_NSCR_lz }, { .palNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NCLR, .charNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NCGR_lz, .scrnNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NSCR_lz }, { .palNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NCLR, .charNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NCGR_lz, .scrnNo = NARC_preview_graphic_preview_graphic_cerulean_cave_day_NSCR_lz }, { .palNo = NARC_preview_graphic_preview_graphic_cerulean_cave_night_NCLR, .charNo = NARC_preview_graphic_preview_graphic_cerulean_cave_night_NCGR_lz, .scrnNo = NARC_preview_graphic_preview_graphic_cerulean_cave_night_NSCR_lz } },
-     .unused3C = 23 }
+                  .scrnNo = NARC_preview_graphic_preview_graphic_dark_cave_r45_night_NSCR_lz,
+            },
+        },
+     .unused3C = 0,
+     },
+    {
+     // Dark Cave, South Side
+        .mapId   = MAP_D42R0102,
+     .unused4 = 0,
+     .files   = {
+            {
+                  .palNo  = NARC_preview_graphic_preview_graphic_dark_cave_r31_morning_NCLR,
+                  .charNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_morning_NCGR_lz,
+                  .scrnNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_morning_NSCR_lz,
+            },
+            {
+                  .palNo  = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NCLR,
+                  .charNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NCGR_lz,
+                  .scrnNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NSCR_lz,
+            },
+            {
+                  .palNo  = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NCLR,
+                  .charNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NCGR_lz,
+                  .scrnNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_day_NSCR_lz,
+            },
+            {
+                  .palNo  = NARC_preview_graphic_preview_graphic_dark_cave_r31_night_NCLR,
+                  .charNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_night_NCGR_lz,
+                  .scrnNo = NARC_preview_graphic_preview_graphic_dark_cave_r31_night_NSCR_lz,
+            },
+        },
+     .unused3C = 0,
+     },
+    {
+     // Sprout Tower
+        .mapId   = MAP_D15R0101,
```

Comment:
not sur why but .mapId's are unaligned here, any way to fix it?

Reply:
> not sur why but .mapId's are unaligned here, any way to fix it?

yes: revert .clang-format